tests/CommunityToolkit.Aspire.Hosting.EventStore.Tests/EventStoreFunctionalTests.cs

```diff
@@ -120,16 +131,18 @@ public async Task WithDataShouldPersistStateBetweenUsages(bool useVolume)
             }
             else
             {
+                //EventStore shutdown can be slightly delayed, so second instance might fail to start when using the same bind mount before shutdown.
+                await Task.Delay(TimeSpan.FromSeconds(5));
```

Comment:
This could incur flakiness.

Related https://github.com/dotnet/aspire/issues/4878